### DISCUSSION TITLE
Theming

### DISCUSSION THREAD

**Discussion by Zyztem** [diced/zipline#231] [Ideas]
Such as the ability to theme the page by changing colors

Could also be extended to include a new admin section which you could upload the favicon, change the website varibles, exc.

**Comment by diced** [comment]
> Such as the ability to theme the page by changing colors

Feel free to create a PR for a theme to be added, unfortunately I will not be adding the ability to theme Zipline like that.

> Could also be extended to include a new admin section which you could upload the favicon

You can change the favicon by changing the default one in the `public/` directory, if you are using docker-compose, it already has the public directory as a volume so it will start working after a restart. It must be a `.ico` and must be named `favicon.ico`.

> change the website varibles, exc.

What do you mean?

**Comment by diced** [comment]
Closing as stale.
